src/util/pointProjection.ts

```diff
@@ -142,15 +142,10 @@ export const usePointWidthsFromIntervals = (scalingMatrix: Matrix, data: number[
 }
 
 
-const applyPixelMargins = (props: ScalingProps) => {
-    const {totalPixelWidth, totalPixelHeight, pixelMargins} = props
-    // const invert = !dontInvertY
-
+const applyPixelMargins = (totalPixelWidth: number, totalPixelHeight: number, pixelMargins?: PartialMargins) => {
```

Comment:
Broke out the individual elements for this, although in retrospect it may not matter since this call isn't being memoized.